Makefile

```diff
@@ -268,20 +268,41 @@ go_ldflags := '${go_ldflags}'
 
 .PHONY: build
 
-build: tidy bin/spire-server bin/spire-agent bin/k8s-workload-registrar bin/oidc-discovery-provider
+build: tidy bin/spire-server bin/spire-agent bin/k8s-workload-registrar bin/oidc-discovery-provider bin/spire-server-static bin/spire-agent-static bin/k8s-workload-registrar-static bin/oidc-discovery-provider-static
 
 define binary_rule
 .PHONY: $1
 $1: | go-check bin/
 	@echo Building $1...
-	$(E)$(go) build $$(go_flags) -ldflags $$(go_ldflags) -o $1 $2
+	$(E)$(go_path) go build $$(go_flags) -ldflags $$(go_ldflags) -o $1 $2
+
+endef
+define binary_rule_static
+.PHONY: $1
+$1: | go-check bin/
+	@echo Building $1... CGO_ENABLED=0 go build $$(go_flags) -ldflags $$(go_ldflags)  -o $1 $2
```

Comment:
Can you remove the build command after `Building $1...` here to be consistent with other build targets' output to stdout?